### PR TITLE
Fix XD96 info.json

### PR DESCRIPTION
**Pull request by fauxpark**
## Description

The info.json for this board doesn't match the layout macros. The numpad + and Enter keys are not where they should be.

## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/xd96/info.json [hunk 1]
```diff
@@ -7,11 +7,231 @@
...
+        {"label":"5", "x":16, "y":3},
+        {"label":"6", "x":17, "y":3},
+        {"label":"+", "x":18, "y":2, "h":2},
+
+        {"label":"Shift", "x":0, "y":4, "w":2.25},
+        {"label":"Z", "x":2.25, "y":4},
+        {"label":"X", "x":3.25, "y":4},
+        {"label":"C", "x":4.25, "y":4},
+        {"label":"V", "x":5.25, "y":4},
+        {"label":"B", "x":6.25, "y":4}
```
Comment (noroadsleft) [inline keyboards/xd96/info.json]: Lost a comma here.

```suggestion
        {"label":"B", "x":6.25, "y":4},
```

Comment (fauxpark) [reply keyboards/xd96/info.json]: That's what I get for doing this by hand...

Comment (noroadsleft) [reply keyboards/xd96/info.json]: I have [quite a bit](https://github.com/qmk/qmk_firmware/pulls?utf8=%E2%9C%93&q=is%3Apr+author%3Anoroadsleft+in%3Atitle+Configurator) of practice at this. :wink:

**Comment by drashna** [comment]
Thanks!

**Pull request merged**